client/src/components/Input/LocationInput.js

```diff
@@ -123,8 +123,15 @@ const LocationInput = ({
       );
       onLocationChange(data.location);
       setSelectedAddress(displaySelectedAddressFromLocation(data.location));
-    } catch {
-      setApiError("Failed sharing location, please input address");
+    } catch(error) {
+      let errMessage = "Failed getting location. " + error.message + ". ";
+      if(error.code === 1) {
+        errMessage += "Please reset your browser's location permissions for this site or enter the address.";
+      }
+      else {
+        errMessage += "Please enter the address or try again.";
```

Comment:
Thanks @EyeWumbo! This will work.

The only thing to fix from my perspective before merging is the formatting. Then we can squash into one commit and merge into staging. 

We have husky+lint-staged that will do this on staged files before committing BUT it doesn't work with docker right now ( see issue #786 ; I had labelled as a good first issue but it might be fairly complicated so I might look into it myself soon)

Until we have that, you can set up other means to apply formatting rules on file save, and/or flag eslint issues. How will vary depending on code editor (feel free to reach out in Slack).

Anyways this is what prettier does when applying formatting on save automatically.

```suggestion
    } catch (error) {
      let errMessage = "Failed getting location. " + error.message + ". ";
      if (error.code === 1) {
        errMessage +=
          "Please reset your browser's location permissions for this site or enter the address.";
      } else {
```